plugins/follow_me/follow_me_impl_test.cpp

```diff
@@ -0,0 +1,13 @@
+#include "follow_me_impl.h"
+#include "dronecore.h"
+#include "global_include.h"
+#include <gtest/gtest.h>
+
+namespace dronecore {
+
+TEST(FollowMeImpl, NoTest)
+{
+    ASSERT_TRUE(true);
```

Comment:
This would be a unit test but if there are none, you can probably leave the file out.

Reply:
Removed, as it is redundant

Reply:
Still looks to be here to me :-)